tests/lax_numpy_reducers_test.py

```diff
@@ -866,5 +866,18 @@ def testCumulativeSumBool(self):
                              dtype=jnp.bool_)
     np.testing.assert_array_equal(np.array([[True], [True], [False]]), out)
 
+  @jtu.sample_product(
+    [dtype for dtype in int_dtypes + unsigned_dtypes],
```

Comment:
This doesn't include `int2` and `uint2` as currently defined – we should either adjust that at the source, or explicitly include those types here.

Also, the test name is `test_low_bitwidth_sum` but the parameterization covers all int types – maybe switch one or the other?

Reply:
I added int2 and uint2 to `all_integers` and `all_unsigned`so that we now test `test_all_integers_sum`